### PR TITLE
Multi distro travis indigo

### PR DESCRIPTION
**Pull request by fmessmer**
fix #56

### CODE REVIEW

**Comment by fmessmer** [comment]
@ipa-rmb 
any objections? are you able to perform a quick functionality test? Thanks!
but let me merge it myself with all the other "Multi distro travis"-PRs...

**Comment by fmessmer** [comment]
kinetic migration is tracked in #66

File: cob_openni2_tracker/launch/create_nite2_link.bash [hunk 1]
```diff
@@ -1,14 +0,0 @@
-#!/bin/bash
```
Comment (ipa-rmb) [inline cob_openni2_tracker/launch/create_nite2_link.bash:1]: Is this file replaced by:
list(APPEND @PROJECT_NAME@_LIBRARIES "${@PROJECT_NAME@_PREFIX}/@CATKIN_PACKAGE_LIB_DESTINATION@/libNiTE2.so")

Comment (fmessmer) [reply cob_openni2_tracker/launch/create_nite2_link.bash:1]: yes, the `libnite2` package no copies both headers and libs into both devel and install space and exports according cmake extras
no need to go through symlink in `~/.ros`

Comment (ipa-rmb) [reply cob_openni2_tracker/launch/create_nite2_link.bash:1]: Nice, thanks!

**Pull request merged**